pkg/rbd/nodeserver_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 The Ceph-CSI Authors.
```

Comment:
2020!

Reply:
The fact that I could've copied that entire header from any other files in this repo indicates that that deserves a PR on its own :)